mxnet-mkl-feedstock/recipe/meta.yaml

```diff
@@ -0,0 +1,101 @@
+package:
+  name: mxnet-mkl
+  version: 1.0.0
+
+source:
+  git_rev: 1.0.0
+  git_url: https://github.com/apache/incubator-mxnet.git
+  patches:
+    - 0001-Make-versions-of-python-dependencies-like-numpy-dete.patch
+    - 0002-Relax-constraints-on-requests-make-graphviz-optional.patch
+
+outputs:
+  - name: libmxnet
+    script: install-libmxnet.sh  # [unix]
+    requirements:
+      host:
+        - mkl-devel {{ mkl }}
+        - opencv
+      build:
+        - {{ compiler('c') }}
+        - {{ compiler('cxx') }}
+        - pkg-config
+        - make
+      # Run exports will take care of opencv, mkl
+    test:
+      commands:
+        - im2rec -h
+        - conda inspect linkages -p $PREFIX $PKG_NAME  # [not win]
+        - conda inspect objects -p $PREFIX $PKG_NAME  # [osx]
+
+  - name: py-mxnet
+    script: install-py-mxnet.sh  # [unix]
+    requirements:
+      host:
+        - libmxnet
+        - cython
+        - python
+        # Numpy's C API is not used
+        - numpy >=1.8.2,<=1.13.3
+        - requests >=2.18.4,<3
+        - {{ pin_subpackage('libmxnet', exact=True) }}
+      build:
+        - {{ compiler('c') }}
+        - {{ compiler('cxx') }}
+      run:
+        - python
+        - {{ pin_subpackage('libmxnet', exact=True) }}
+        # https://github.com/apache/incubator-mxnet/pull/9169
+        - numpy >=1.8.2,<=1.13.3
+        - requests >=2.18.4,<3
+        # This is only required for visualization and hence optional.
+        # We need a new build of graphviz otherwise it downgrades opencv
+        #- python-graphviz
+    test:
+      imports:
+        - mxnet
+
+  - name: mxnet-cpp
```

Comment:
What is this package? IMHO there should be no packages ending with `-cpp`. For the following reasons:

1. `conda-forge` thought it stood for C++, but it does not. It stands for C preprocessor.
2. We use prefixes not suffixes to indicate package namespaces (though I'm not *really* sure that's your intention?)
3. It seems this is just C headers which indicates that you might mean `cpp` to stand for `C preprocessor`? Regardless, due to CF, these waters are too muddied.
4. Mostly, why bother? Headers are tiny and should be bundled with `libmxnet` where they are most useful and can most reasonably be expected to be found.

Reply:
The upstream folks maintain it as a separate package.
https://github.com/apache/incubator-mxnet/tree/master/cpp-package

What should we call it? Should we still merge it with libmxnet?

Reply:
Hmm.. the headers are inside mxnet-cpp subfolder anyway, so I'll just combine them in libmxnet.